EntityLayer/Entities/FloorMap.cs

```diff
@@ -11,5 +11,9 @@ public class FloorMap
         public int Id { get; set; }
         public string Name { get; set; }
         public string Image { get; set; }
+
+        //public virtual ICollection<Asset> Assets { get; set; } = new List<Asset>();
```

Comment:
comment?

Reply:
This code will be used after creating the AssetZoneHistory class.